Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -4438,4 +4442,25 @@ RefPtr<CSSStyleValue> Parser::parse_filter_value_list_value(TokenStream<Componen
     return FilterValueListStyleValue::create(move(filter_value_list));
 }
 
+RefPtr<CSSStyleValue> Parser::parse_view_transition_name_value(TokenStream<ComponentValue>& tokens)
+{
+    // none | <custom-ident>
+    tokens.discard_whitespace();
+    {
+        auto transaction = tokens.begin_transaction();
+
+        // The values 'none' and 'auto' are excluded from <custom-ident> here.
+        // Note: Only auto is excluded here since none isn't parsed differently.
```

Comment:
There's a subtle trap here: `<custom-ident>`s are case-sensitive, so whereas `nOnE` would parse as a `none` keyword, it would parse as a `nOnE` custom ident. This will make serialization wrong, and the string comparison in `ComputedProperties::view_transition_name()` too.

Reply:
Fixed that by checking for case-insensitive "none" and then returning a lower-case "none" CustomIdentStyleValue.